data/interfaces/default/base.html

```diff
@@ -20,12 +20,130 @@
     ${next.headIncludes()}
 
     <link rel="icon" type="image/x-icon" href="interfaces/default/images/favicon.ico"/>
-    <!-- touch icons -->
     <link rel="shortcut icon" href="interfaces/default/images/favicon.png">
-    <link rel="apple-touch-icon" href="interfaces/default/images/icon_iphone.png">
-    <link rel="apple-touch-icon" sizes="72x72" href="interfaces/default/images/icon_ipad.png">
-    <link rel="apple-touch-icon" sizes="114x114" href="interfaces/default/images/icon_iphone@2x.png">
-    <link rel="apple-touch-icon" sizes="144x144" href="interfaces/default/images/icon_ipad@2x.png">
+
+    <!-- Allow web app to be run in full-screen mode. -->
+    <meta name="apple-mobile-web-app-capable"
+          content="yes">
+    <!-- Configure the status bar. -->
+    <meta name="apple-mobile-web-app-status-bar-style"
+          content="black">
+    <!-- Set the viewport. -->
+    <meta name="viewport"
+          content="initial-scale=1">
+    <!-- Disable automatic phone number detection. -->
+    <meta name="format-detection"
+          content="telephone=no">
+
+    <!-- ICONS -->
+    <!-- IE10 icon --> 
+    <meta name="PlexPy" content="PlexPy"/>
+    <meta name="msapplication-TileColor" content="#000000"/>
+    <meta name="msapplication-square70x70logo" content="tiny.png"/>
+    <meta name="msapplication-square150x150logo" content="square.png"/>
+    <meta name="msapplication-wide310x150logo" content="wide.png"/>
+    <meta name="msapplication-square310x310logo" content="large.png"/>
+    <!-- Android >M39 icon -->
+    <link rel="icon" sizes="48x48"href="interfaces/default/images/res/android/48.png">
+    <link rel="icon" sizes="72x72"href="interfaces/default/images/res/android/72.png">
+    <link rel="icon" sizes="96x96"href="interfaces/default/images/res/android/96.png">    
+    <link rel="icon" sizes="144x144"href="interfaces/default/images/res/android/144.png">
+    <link rel="icon" sizes="192x192"href="interfaces/default/images/res/android/192.png">
+    <link rel="icon" sizes="512x512"href="interfaces/default/images/res/android/512.png">
+    <!-- iPad retina icon -->
```

Comment:
To specify multiple icons for different device resolutions it's necessary to add These lines. (i think so)